pkgs/os-specific/linux/mba6x_bl/default.nix

```diff
@@ -0,0 +1,33 @@
+{ fetchFromGitHub, kernel, stdenv }:
+
+with stdenv.lib;
+
+let version = "2016-02-12";
+    pkgName = "mba6x_bl";
```

Comment:
Just nitpicking: these variables are only used once, so there is no need to extra define them here.